src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx

```diff
@@ -792,42 +795,55 @@ Use [Elastic.Clients.Elasticsearch](https://www.nuget.org/packages/Elastic.Clien
           <td>
             v9.7.0
           </td>
+          <td>
+            2.0.10, 2.0.12, 2.0.13, 2.0.14
+          </td>
+
         </tr>
 
         <tr>
           <td>
             Serilog
           </td>
           <td>
-            v2.5
+            v2.5.0
```

Comment:
Could we remove the leading `v` on the version numbers everywhere in this document?